.github/workflows/namespace-notifier.yml

```
@@ -0,0 +1,26 @@
+name: Buildpack Registry Namespace Notifier
+on: 
+  push:
+    branches:
+      - main-test
+
+jobs:
+  email-maintainers:
+    name: Email Maintainers
+    runs-on: ubuntu-latest
+    steps:
+    - name: Send mail
+      uses: dawidd6/action-send-mail@v2
+      with:
+        server_address: smtp.gmail.com
+        server_port: 465
+        username: ${{ secrets.MAIL_USERNAME }}
+        password: ${{ secrets.MAIL_PASSWORD }}
+        subject: GITHUB ACTIONS ${{ github.repository }} updated !!!
+        body: Maintainers, there are some changes in ${{ github.repository }} repository.
+        to: ${{ secrets.MAIL_RECEIVER }}
```

```suggestion
        to: ${{ secrets.CNB_MAINTAINERS_EMAIL }}
```

Let's change the name of this var so it's obvious that it's not related to the sender vars